icicle/CMakeLists.txt

```diff
@@ -18,6 +18,11 @@ endif()
 # Print the selected build type
 message(STATUS "Build type: ${CMAKE_BUILD_TYPE}")
 
+# Prevent build if both SANITIZE and CUDA_BACKEND are enabled
+if(SANITIZE AND CUDA_BACKEND) 
```

Comment:
Why SANITIZE AND CUDA_BACKEND couldn't both be true?
What happens in tests that CPU result if compared against GPU?

Reply:
sanitizer and cuda does not work together